dallinger/command_line.py

```diff
@@ -724,12 +729,13 @@ def export(app, local, no_scrub):
 @click.option('--app', default=None, help='ID of the deployed experiment')
 def logs(app):
     """Show the logs."""
-    if app is None:
-        raise TypeError("Select an experiment using the --app flag.")
-    else:
+    verify_id(app)
+    try:
         subprocess.check_call([
             "heroku", "addons:open", "papertrail", "--app", app_name(app)
         ])
+    except subprocess.CalledProcessError as e:
```

Comment:
This CalledProcessError will be raised whenever the call to the Heroku CLI returns a non-zero exit status, not just when the experiment ID is invalid (e.g., if there's a typo in `heroku`). I think it'll print a useful error message via the CLI anyways. So I think I'd recommend removing the try/except here and call `check_call` directly.